ui/mods/msu/keybinds/keybinds_system.js

```diff
@@ -0,0 +1,81 @@
+MSU.Keybinds = {
+	KeybindsByMod : {},
+	PressedKeys : {},
+	addKeybindFromSQ : function( _modID, _id, _keyCombinations )
+	{
+		if (!(_modID in this.KeybindsByMod))
+		{
+			this.KeybindsByMod[_modID] = {};
+		}
+		this.KeybindsByMod[_modID][_id] = new MSU.Keybind(_modID, _id, _keyCombinations);
+	},
+
+	removeKeybind : function( _modID, _id )
+	{
+		delete this.KeybindsByMod[_modID][_id];
+		if (Object.keys(this.KeybindsByMod[_modID]).length === 0)
+		{
+			delete this.KeybindsByMod[_modID];
+		}
+	},
+
+	getPressedKeysAsString : function( _excludeKey )
+	{
+		var key = '';
+		Object.keys(this.PressedKeys).forEach(function(pressedKeyID)
+		{
+			if (_excludeKey != pressedKeyID)
+			{
+				key += pressedKeyID + "+";
+			}
+		});
+		return key;
+	},
+
+	updatePressedKeys : function( _key, _release )
+	{
+		if (!_release)
```

Comment:
I think in JS it is very sensible to explicitly compare bools, ie `release === false`; truthiness is messy in JS.